core/domain/user_services.py

```diff
@@ -361,20 +586,48 @@ def get_or_create_user(user_id, email):
 
 
 def get_username(user_id):
+    """Gets username corresponding to the given user_id.
+
+    Args:
+        user_id: str. The user id.
+
+    Returns:
+        str. Username corresponding to the given user_id.
+    """
     if user_id == feconf.MIGRATION_BOT_USER_ID:
         return feconf.MIGRATION_BOT_USERNAME
     else:
         return get_user_settings(user_id, strict=True).username
 
 
 def get_usernames(user_ids):
+    """Gets usernames corresponding to the given user_ids.
+
+    Args:
+        user_ids: list(str). The list of user_ids to get UserSettings
+            domain objects for.
+
+    Returns:
+        list. Containing usernames based on given user_ids.
+        If a user_id does not exist, the corresponding entry in the
+        returned list is None.
+    """
     users_settings = get_users_settings(user_ids)
     return [us.username if us else None for us in users_settings]
 
 
 # NB: If we ever allow usernames to change, update the
 # config_domain.BANNED_USERNAMES property.
 def set_username(user_id, new_username):
+    """Sets username to new_username of User with given user_id.
```

Comment:
Wording is a bit clumsy. Perhaps: Updates the username of the user with the given user_id.